core/runtime/src/main/java/io/quarkiverse/langchain4j/data/AiStatsMessage.java

```diff
@@ -0,0 +1,54 @@
+package io.quarkiverse.langchain4j.data;
+
+import java.util.List;
+
+import dev.langchain4j.agent.tool.ToolExecutionRequest;
+import dev.langchain4j.data.message.AiMessage;
+import dev.langchain4j.internal.ValidationUtils;
+import dev.langchain4j.model.output.TokenUsage;
+
+public class AiStatsMessage extends AiMessage {
```

Comment:
We'll definitely need some Javadoc on what this tracks

Reply:
done